### PR TITLE
created multiple employee cards using json objects in javascript, gupl is used for styling

### PR DESCRIPTION
**Pull request by sansadev**

### CODE REVIEW

File: lesson03/assignment/data/data.json [hunk 1]
```diff
@@ -51,4 +51,31 @@
        "HTML", "CSS", "JavaScript", "Java", "Spring"
     ]
   }
- ]
\ No newline at end of file
+ ];
+
+const employee-info-all = template-hook.map(function(emp) {
```
Comment (cherimarie) [inline lesson03/assignment/data/data.json]: Javascript naming convention is camelCase- no spaces, no hyphens, start with lower case letter then each additional word starts with capital letter.

File: lesson03/assignment/data/data.json [hunk 2]
```diff
@@ -51,4 +51,31 @@
...
+  <div class="linkedin-img-logo"><img src="img/linkedin.svg" alt="linkedin-logo">
+  <span class="linkedin-email">${emp.linkedInUrl}</span></div>
+
+  </div>
+  </div>
+  </section>
+  `;
+  });
+
+  $(".template-hook").append(employee-info-all);
```
Comment (cherimarie) [inline lesson03/assignment/data/data.json]: There shouldn't be any javascript in this file, it's just for holding the JSON data.

File: lesson03/assignment/index.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  <span class="linkedin-email">${emp.linkedInUrl}</span></div>
+
+  </div>
+  </div>
+
+  `
+  return template;
+  });
+
+  //$(".template-hook").append(employeeInfoAll);
```
Comment (cherimarie) [inline lesson03/assignment/index.js]: No need to keep commented out code.

File: lesson03/assignment/index.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    "major": "Trash Talking",
+    "email": "mJordan@bulls.com",
+    "linkedInUrl": "mJordas.linkedinprofile.com",
+    "codeLanguages": [
+       "HTML", "CSS", "JavaScript", "Java", "Spring"
+    ]
+  }
+ ];
+
+const employeeInfoAll = templateHook.map(function(emp) {
```
Comment (tylervanbrocklin) [inline lesson03/assignment/index.js:56]: Your map function looks great! Nice work on this one.

File: lesson03/assignment/index.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  </div>
+
+  `
+  return template;
+  });
+
+ 
+  document.getElementById("template-hook").innerHTML = employeeInfoAll;
+
+console.log(employeeInfoAll);
```
Comment (tylervanbrocklin) [inline lesson03/assignment/index.js:85]: Now that you've gotten everything working, it's generally good practice to remove `console.log()` you used for debugging purposes.

**Pull request merged**